client/src/components/Ratings/sub-components/UploadImageModal.jsx

```diff
@@ -44,76 +43,72 @@ export default function UploadAndDisplayImage({
   };
 
   // EVENT HANDLERS //
-  const closeModal = (e) => {
-    if (e.key === 'Escape' || e.type === 'Click') {
+  const closeModal = () => {
+    document.querySelector('.write-review-modal').classList.add('return');
+    setTimeout(() => {
       setImageUploadModal(false);
-    }
-    setImageUploadModal(false);
+    }, 200);
+    // setTimeout(() => {
+    // setImageUploadModal(false);
+    // document.querySelector('.write-review-modal').classList.remove('exiting');
+    // }, 200);
```

Comment:
Delete the lines if you don't need. Otherwise, put a comment reason here when you make pr.

Reply:
Removed and cleaned up file